R/block_cv.R

```diff
@@ -34,13 +34,17 @@
 #'   and an identification variable `id`.
 #'
 #' @examples
-#' data(Smithsonian, package = "modeldata")
-#' smithsonian_sf <- sf::st_as_sf(Smithsonian,
-#'                                coords = c("longitude", "latitude"),
-#'                                # Set CRS to WGS84
-#'                                crs = 4326)
+#' if (requireNamespace("modeldata", quietly = TRUE)) {
```

Comment:
Instead of using `if()` this like, use `@examplesIf rlang::is_installed("modeldata"))` like this:
https://github.com/rstudio/vetiver-r/blob/main/R/ptype.R#L34
It can totally replace the `@examples` line, line 36.

Reply:
:open_mouth: cool!